classes/PodsData.php

```diff
@@ -2534,7 +2542,12 @@ public static function query_field ( $field, $q, $pod = null, &$params = null )
             	$field_query = $field_cast . ' ' . $field_compare . ' "' . $field_value . '"';
 			}
 		}
-        elseif ( in_array( $field_compare, array( 'IN', 'NOT IN' ) ) ) {
+        elseif ( in_array( $field_compare, array( 'IN', 'NOT IN', 'ALL' ) ) ) {
+            if ( $field_compare == 'ALL' ) {
+                $field_compare = 'IN';
+                $params->having[] = 'count(DISTINCT ' . $field_cast . ') = ' . count( $field_value );
+                $params->groupby[] = '`t`.`ID`';
+            }
```

Comment:
Please keep a line between each `if` statement, so it just needs one added below this.